### DISCUSSION TITLE
Is anyone here interested in a MPD Discord?

### DISCUSSION THREAD

**Discussion by jcorporation** [MusicPlayerDaemon/MPD#2128] [General]
As the title says.

**Comment by MaxKellermann** [comment]
I once tried to register with Discord, but they want my phone number which I'm not willing to give them. Not for me.

**Comment by Huecuva** [comment]
I already have a Discord account so the phone number thing is moot for me. If I can actually get responses in the Discord then sure, since nobody seems to be answering my question here.

**Comment by ghost** [comment]
I'd be more interested in a Signal or Matrix group, but I'll take what I can get.

If it is a Discord server though then there needs to be a music bot that runs an MPD stream!
